library/ecjpake.c

```diff
@@ -726,6 +752,10 @@ int mbedtls_ecjpake_derive_secret( mbedtls_ecjpake_context *ctx,
     mbedtls_mpi m_xm2_s, one;
     unsigned char kx[MBEDTLS_ECP_MAX_BYTES];
     size_t x_bytes;
+    ECJPAKE_VALIDATE_RET( ctx   != NULL );
```

Comment:
Not a blocker: please leave a blank line after the block of declarations.

Reply:
Fixing.